nebula_common/include/nebula_common/continental/continental_ars548.hpp

```diff
@@ -246,13 +248,24 @@ constexpr int object_filter_properties_num = 24;
 constexpr int max_detections = 800;
 constexpr int max_objects = 50;
 
+constexpr int measurement_status_measured = 0;
+constexpr int measurement_status_predicted = 1;
+constexpr int measurement_status_new = 2;
+constexpr int measurement_status_invalid = 255;
+
+constexpr int movement_status_dynamic = 0;
+constexpr int movement_status_static = 1;
+constexpr int movement_status_invalid = 255;
+
```

Comment:
This is probably done to keep ROS stuff out of `nebula_common`, right? The problem I see with this is that it can easily become de-synced with the message definition (it's already de-synced at this moment).

Also, conversion code becomes necessary in other places such as here:

https://github.com/knzo25/nebula/blob/2a5e9cf24f87add12601ebdb5844e665b9982ec0/nebula_ros/src/continental/continental_ars548_decoder_wrapper.cpp#L508-L539

I think giving in and letting ROS message definitions into here would be preferable due to the concerns above. Alternatively, moving this to `nebula_ros` (where using ROS messages is obviously fine) and removing the code that became redundant by that would be an even better option.

Reply:
The conversion is necessary because continental and autoware have their own definitions and they do not need to match, so the continental definitions are needed somewhere in the codebase, and I believe this is the correct place.

For example, probabilities in continental messages are 0-100 (with the raw prefix to remark the fact that they are not normalized)

Reply:
Ah, indeed. I only remembered the discussion for radar messages where I suggested changing invalid from `255` to `0`. I see where this was coming from now :innocent: 

In that case, I think this is fine as-is.